src/NLog/MessageTemplates/CaptureType.cs

```diff
@@ -38,6 +38,10 @@ namespace NLog.MessageTemplates
     /// </summary>
     public enum CaptureType : byte
     {
+        /// <summary>
+        /// Not decided
+        /// </summary>
+        Unknown,
```

Comment:
what do you think about a  nullable CaptureType? Or does this have to much impact?

Reply:
I prefer to have Unknown